plugins/kubernetes-backend/src/service/KubernetesFanOutHandler.ts

```diff
@@ -203,24 +215,22 @@ export class KubernetesFanOutHandler {
     );
 
     const labelSelector: string =
```

Comment:
What would it take to make the label presence optional?  It's easy to forget applying the correct label when creating resources, so it's causing us a lot of headaches.

Reply:
@mklanjsek Could you write this up into an issue? Would be good to get input from people.